docs/pages/tutorial/create-a-modal.mdx

```diff
@@ -558,23 +558,23 @@ Create a **EmojiList.tsx** file inside the **components** directory and add the
 ```tsx components/EmojiList.tsx
 import { useState } from 'react';
 import { StyleSheet, FlatList, Platform, Pressable } from 'react-native';
-import { Image } from 'expo-image';
+import { Image, type ImageSource } from 'expo-image';
 
 type Props = {
-  onSelect: (image: string) => void;
+  onSelect: (image: ImageSource) => void;
   onCloseModal: () => void;
 };
 
-export default function EmojiList({ onSelect, onCloseModal }: Props) {
-  const [emoji] = useState([
-    require('../assets/images/emoji1.png'),
-    require('../assets/images/emoji2.png'),
-    require('../assets/images/emoji3.png'),
-    require('../assets/images/emoji4.png'),
-    require('../assets/images/emoji5.png'),
-    require('../assets/images/emoji6.png'),
-  ]);
+const emoji: ImageSource[] = [
+  require('../assets/images/emoji1.png'),
+  require('../assets/images/emoji2.png'),
+  require('../assets/images/emoji3.png'),
+  require('../assets/images/emoji4.png'),
+  require('../assets/images/emoji5.png'),
+  require('../assets/images/emoji6.png'),
+];
```

Comment:
```suggestion
export default function EmojiList({ onSelect, onCloseModal }: Props) {
  const [emoji] = useState<ImageSource[]>([
    require("../assets/images/emoji1.png"),
    require("../assets/images/emoji2.png"),
    require("../assets/images/emoji3.png"),
    require("../assets/images/emoji4.png"),
    require("../assets/images/emoji5.png"),
    require("../assets/images/emoji6.png"),
  ]);
```

The `ImageSource[]` can be passed directly to the `useState`. I'm avoiding the change here so as not to differ from the video tutorial that is a companion to this chapter.